sdks/go/pkg/beam/options/jobopts/options.go

```diff
@@ -31,6 +31,18 @@ import (
 	"github.com/apache/beam/sdks/go/pkg/beam/log"
 )
 
+func init() {
+	flag.Var(&SdkHarnessContainerImageOverrides,
+		"sdk_harness_container_image_override",
+		"Overrides for SDK harness container images. Could be for the "+
+			"local SDK or for a remote SDK that pipeline has to support due "+
+			"to a cross-language transform. Each entry consist of two values "+
+			"separated by a comma where first value gives a regex to "+
+			"identify the container image to override and the second value "+
+			"gives the replacement container image. Multiple entries can be "+
+			"specified by using this flag multiple times.")
```

Comment:
Just confirming, this part matches the semantics of the equivalent flag in the other SDKs?

Reply:
I haven't been able to find the documentation for the Java version of this flag (despite being relatively sure it exists, based on [this line](https://github.com/apache/beam/blob/76c49ac276fc804ccc058c901dc6cb9e063b2529/runners/google-cloud-dataflow-java/build.gradle#L345)), but the Python version works identically, it requires you to use the flag once per entry.